src/main/java/mil/dds/anet/utils/Utils.java

```diff
@@ -336,4 +345,60 @@ public static boolean isEmailWhitelisted(String email, List<String> whitelistDom
 
     return isWhitelistedEmail || isValidWildcardDomain;
   }
+
+  /**
+   * Resizes an image
+   * 
+   * @param imageBase64 The image as a Base64 string
+   * @param width The desired output width
+   * @param height The desired output height
+   * @param imageFormatName The desired output format (png, jpg)
+   * @return The resized image as a Base64 string
+   * @throws IOException When the conversion fails
+   */
+  public static String resizeImageBase64(String imageBase64, int width, int height,
+      String imageFormatName) throws IOException {
+
+    if (imageBase64 == null || imageBase64.isEmpty()) {
+      return imageBase64;
+    }
+
+    // From Base64-string to BufferedImage
+    final BufferedImage imageBytes = convert(imageBase64);
+
+    // Resizing
+    final BufferedImage thumbnail =
+        Scalr.resize(imageBytes, Method.QUALITY, Mode.AUTOMATIC, width, height);
```

Comment:
Isn't `Method.Automatic` is likely more adapted? In most cases we will be requesting sizes that do not result in aliasing (i.e. 128,64,32... ) so we want to benefit from fast conversions.

Reply:
I was testing whether the performance hit was worth it, and whether a lower quality was acceptable. AUTOMATIC probably gives the best balance.